openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -323,6 +330,21 @@ def test_create_account(self):
             'extended_profile': [],
         })
 
+    @override_waffle_flag(PASSWORD_UNICODE_NORMALIZE, active=True)
+    def test_normalize_password(self):
+        # Test password normalization on account creation.
+        create_account(self.USERNAME, unicodedata.normalize('NFKD', u'Ṗŕệṿïệẅ Ṯệẍt'), self.EMAIL)
```

Comment:
To make it more clear, I would explicitly mention here that you are purposefully normalizing it here to the "incorrect" normalization method. Took me a second to figure that out. Or write it in a docstring for the method.